imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js

```diff
@@ -55,7 +106,25 @@ Template.searchModal.onCreated(function () {
        * Product Search
        */
       if (searchCollection === "products") {
-        const productResults = ProductSearch.find().fetch();
+        let productResults = ProductSearch.find().fetch();
+        // filter product by price if the filter array is not null or all
+        if (!["null", "all"].includes(priceQuery) && priceQuery) {
```

Comment:
you may want to have all of this as `priceQuery && !["null", "all"].includes(priceQuery)`
why? means you check if priceQuery is defined before checking for the value in the array